lib/src/main/java/com/sensirion/libble/devices/Peripheral.java

```diff
@@ -741,6 +741,27 @@ public boolean areAllServicesReady() {
      * {@inheritDoc}
      */
     @Override
+    public boolean synchronizeAllDeviceServices() {
+        final List<BleService> servicesToSynchronize = new LinkedList<>(mServices);
+        // Synchronize all the services considering their priorities
+        Collections.sort(servicesToSynchronize, SERVICE_PRIORITY_COMPARATOR);
+        for (final BleService service : servicesToSynchronize) {
+            if (service.isServiceReady()) {
```

Comment:
I did this for following the Sensirion convention: `Avoid using negative if conditions`

I can use negations on this type of cases on the future?